dbtwiz/generate/model.py

```diff
@@ -0,0 +1,156 @@
+import os
+import yaml
+
+from dbtwiz.interact import (
+    input_text,
+    select_from_list,
+    multiselect_from_list,
+    autocomplete_from_list,
+)
+
+from dbtwiz.logging import info, warn, error, fatal
+
+from dbtwiz.model import (
+    Group,
+    Project,
+    layer_choices,
+    materialization_choices,
+    access_choices,
+    domains_for_layer,
+    model_base_path,
+)
+
+
+def generate_model(quick: bool):
+    """Generate new dbt model"""
+
+    project = Project()
+
+    try:
+        layer = select_from_list(
+            "Select model layer",
+            layer_choices())
+
+        domain = autocomplete_from_list(
+            "Which domain does your model belong to",
+            domains_for_layer(layer),
+            must_exist=False,
+            allow_blank=False)
+
+        while True:
+            name = input_text(
+                "What is the name of your model")
```

Comment:
Since the model doesn't exist (yet), perhaps change the prompt to something like "Give a name for your new model" or "What should the name of your model be"